[2022-02-10 15:44:31,369] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-10 15:44:31,381] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-10 15:44:31,382] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:44:31,384] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-10 15:44:31,385] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:44:31,404] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-04-01 00:00:00+00:00
[2022-02-10 15:44:31,419] {standard_task_runner.py:52} INFO - Started process 1356 to run task
[2022-02-10 15:44:31,428] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-04-01T00:00:00+00:00', '--job-id', '894', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpyldu5fqw', '--error-file', '/tmp/tmparitqy0d']
[2022-02-10 15:44:31,430] {standard_task_runner.py:77} INFO - Job 894: Subtask download_dataset_task
[2022-02-10 15:44:31,551] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:44:31,625] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:44:31,650] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-01T00:00:00+00:00
[2022-02-10 15:44:31,651] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:44:31,653] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-04.csv > /opt/***/yellow_tripdata_2019-04.csv']
[2022-02-10 15:44:31,674] {subprocess.py:85} INFO - Output:
[2022-02-10 15:45:37,710] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:45:37,917] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190401T000000, start_date=20220210T154431, end_date=20220210T154537
[2022-02-10 15:45:38,006] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:45:38,110] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:49:10,108] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-10 16:49:10,135] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-10 16:49:10,139] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:49:10,141] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-10 16:49:10,142] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:49:10,169] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-04-01 00:00:00+00:00
[2022-02-10 16:49:10,184] {standard_task_runner.py:52} INFO - Started process 1286 to run task
[2022-02-10 16:49:10,203] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-04-01T00:00:00+00:00', '--job-id', '1002', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmphscupt6s', '--error-file', '/tmp/tmpg0w_mlmu']
[2022-02-10 16:49:10,205] {standard_task_runner.py:77} INFO - Job 1002: Subtask download_dataset_task
[2022-02-10 16:49:10,336] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:49:10,410] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:49:10,448] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-01T00:00:00+00:00
[2022-02-10 16:49:10,451] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:49:10,453] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-04.csv > /opt/***/yellow_tripdata_2019-04.csv']
[2022-02-10 16:49:10,475] {subprocess.py:85} INFO - Output:
[2022-02-10 16:50:13,701] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:50:13,849] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190401T000000, start_date=20220210T164910, end_date=20220210T165013
[2022-02-10 16:50:13,891] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:50:13,953] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
